src/donor_dashboard/views.py

```diff
@@ -28,9 +28,7 @@
 @login_required
 def get_org_info(request, organization_id):
     organization = Organization.objects.get(organization_id=organization_id)
-    donations = Donation.objects.filter(
-        organization_id=organization.organization_id, active=True
-    )
+    donations = Donation.objects.filter(organization_id=organization.organization_id)
```

Comment:
Just wanted to confirm- removing active filter would list all the deleted donations as well right? If yes, then I think the stats would get messed up. Great if thats not the case!

Reply:
Ah yes, or perhaps it could get fixed on the stats level to only use active donations.

Reply:
@arsalananwar11 only active donations will be displayed on the donation list

https://github.com/gcivil-nyu-org/wed-fall24-team5/blob/56eab825d7699dbfde4abf60b290f2a130ba373e/src/donor_dashboard/templates/donor_dashboard/manage_organization.html#L163

**but** you are right that this will mess up statistics, however it is not because of this function - it's because of the `filter_statistics` function. I added the active filter back into the `filter_statistics` view to only include active donations.

https://github.com/gcivil-nyu-org/wed-fall24-team5/blob/56eab825d7699dbfde4abf60b290f2a130ba373e/src/donor_dashboard/views.py#L307-L309

Reply:
Updated: 
https://github.com/gcivil-nyu-org/wed-fall24-team5/blob/ea91989f7da647b5d52d69aaa4945ddc9389c4a4/src/donor_dashboard/views.py#L307-L311

Reply:
great catch, thank you for the review!!

Reply:
(also if my understanding is wrong please let me know!)

Reply:
@arsalananwar11 you're right - i was getting confused between organization active status and donation active status !

Reply:
ok @arsalananwar11 ready for re-review !